zerver/tests/test_signup.py

```diff
@@ -3108,38 +3112,39 @@ def _assert_redirected_to(self, result: HttpResponse, url: str) -> None:
 
     def test_bad_email_configuration_for_accounts_home(self) -> None:
         """
-        Make sure we redirect for SMTP errors.
+        Make sure we redirect for EmailNotDeliveredException.
         """
         email = self.nonreg_email("newguy")
 
         smtp_mock = patch(
             "zerver.views.registration.send_confirm_registration_email",
-            side_effect=smtplib.SMTPException("uh oh"),
+            side_effect=EmailNotDeliveredException,
         )
 
         with smtp_mock, self.assertLogs(level="ERROR") as m:
             result = self.client_post("/accounts/home/", {"email": email})
 
         self._assert_redirected_to(result, "/config-error/smtp")
-
-        self.assertEqual(m.output, ["ERROR:root:Error in accounts_home: uh oh"])
+        # EmailNotDeliveredException doesn't output an additional messsage
```

Comment:
I don't think this comment adds any value here, because there's no "additional message" to have never been talking about.  Let's drop it?